apps/macos/app.json

```diff
@@ -1,4 +1,7 @@
 {
   "name": "FluentTester-macos",
-  "displayName": "FluentTester-macos"
+  "displayName": "FluentTester-macos",
+  "components": [
```

Comment:
What is this doing?

Reply:
`components`? It's to declare the apps that you registered with `AppRegistry`. The list is used to populate the React menu so you can switch between them. There's only one app in the FluentUI test app so it's not very useful here. You can see it in use here:

![react-native-test-app-macos](https://user-images.githubusercontent.com/4123478/83296463-24835b00-a1f1-11ea-86bc-ca4aebffc863.gif)